spaceship/lib/spaceship/tunes/app_review_attachment.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'tunes_base'
+
+module Spaceship
+  module Tunes
+    # Represents an image hosted on App Store Connect. Used for app store review attachment file.
+    class AppReviewAttachment < TunesBase
+      HOST_URL = "https://iosapps-ssl.itunes.apple.com/itunes-assets"
+
+      attr_accessor :asset_token
+
+      attr_accessor :original_file_name
+
+      attr_accessor :url
+
+      attr_accessor :type_of_file
+
+      attr_mapping(
+        'assetToken' => :asset_token,
+        'fileType' => :type_of_file,
+        'url' => :url,
+        'name' => :original_file_name
+      )
+
+      def reset!(attrs = {})
+        update_raw_data!(
+          {
+            asset_token: nil,
+            type_of_file: nil,
+            url: nil,
+            original_file_name: nil
+          }.merge(attrs)
+        )
+      end
+
+      def setup
+        # Since September 2015 we don't get the url any more, so we have to manually build it
+        self.url = "#{HOST_URL}/#{self.asset_token}"
```

Comment:
I think it might be better to override the `url` method instead of using the `setup` hook which is a little janky IMO.
```ruby
def url
  "#{HOST_URL}/#{self.asset_token}"
end
```

Reply:
This is modelled after https://github.com/fastlane/fastlane/blob/1af33c0884e783867bdc07a324280ec6bef124ad/spaceship/lib/spaceship/tunes/app_image.rb#L37-L40

Reply:
Ah, I will support consistency over the slight code quality improvement.